src/YoutubeAtom.tsx

```diff
@@ -76,11 +248,117 @@ export const YoutubeAtom = ({
     title = 'YouTube video player',
     duration,
     origin,
+    gaEventEmitter,
+    ophanEventEmitter,
 }: YoutubeAtomType): JSX.Element => {
     const embedConfig =
         adTargeting && JSON.stringify(buildEmbedConfig(adTargeting));
     const originString = origin ? `&origin=${origin}` : '';
     const iframeSrc = `https://www.youtube.com/embed/${videoMeta.assetId}?embed_config=${embedConfig}&enablejsapi=1${originString}&widgetid=1&modestbranding=1`;
+
+    const [hasUserLaunchedPlay, setHasUserLaunchedPlay] = useState<boolean>(
+        false,
+    );
+    const [isPlayerReady, setIsPlayerReady] = useState<boolean>(false);
+    const [player, setPlayer] = useState<YT.Player | null>(null);
+
+    const loadVideo = () => {
+        setPlayer(
+            new window.YT.Player(`${videoMeta.assetId}`, {
+                videoId: `${videoMeta.assetId}`,
+                events: {
+                    onReady: () => setIsPlayerReady(true),
+                    // Issue with setting events on Youtube object (refer to useEffect below)
+                    // https://stackoverflow.com/a/17078152
+                    // onPlayerStateChange,
+                },
+                playerVars: {
+                    mute: 1,
+                    autoplay: 1,
```

Comment:
```suggestion

                playerVars: {
                    mute: 1,
                    autoplay: 1, // Enabling autoplay because the video loads when a user clicks so when the youtube iframe loads, we want the video to load
```